### PR TITLE
tests: add regression tests for issues fixed in the past

### PR DESCRIPTION
**Pull request by nurmukhametov**
This PR adds several tests for issues that had been fixed in previous releases but issues have been opened until now.

### CODE REVIEW

File: tests/lit-tests/1191.ispc [hunk 1]
```diff
@@ -0,0 +1,6 @@
+// RUN: not %{ispc} --target=host --nostdlib --nowrap -o - --emit-llvm-text %s 2>&1 | FileCheck %s
```
Comment (aneshlya) [inline tests/lit-tests/1191.ispc:1]: The issue number was 1197.

File: tests/lit-tests/1238.ispc [hunk 2]
```diff
@@ -0,0 +1,52 @@
+// RUN: %{ispc} --arch=x86-64 --pic -O3 -DINLINE=inline --target=sse4-i32x8,avx1-i32x8,avx2-i32x8 --nostdlib --nowrap -o %t.ll --emit-llvm-text %s 2>&1 | FileCheck %s
```
Comment (aneshlya) [inline tests/lit-tests/1238.ispc:1]: I'd simplify this test. `-DINLINE` can be replaced with explicit `inline` specifier. Also NoisePerm can be reduced in size.

**Review by aneshlya — COMMENTED** [review]
Please close #1248 as well.

**Comment by nurmukhametov** [comment]
> Please close #1248 as well.

I am not sure that it should be closed because it consists of two parts. The first one is about compiler crash on some code. The second one is about the compiler considering some code as wrong. The first one is fixed, the second one is not.

File: tests/lit-tests/1191.ispc (hunk 1 above)
Comment (nurmukhametov) [reply tests/lit-tests/1191.ispc:1]: Indeed, fixed.

File: tests/lit-tests/905.ispc [hunk 3]
```diff
@@ -0,0 +1,4 @@
+// RUN: not %{ispc} --target=host --nostdlib --nowrap -o - --emit-llvm-text %s 2>&1 | FileCheck %s
```
[the remark this answers is not in the document]
Comment (nurmukhametov) [reply tests/lit-tests/905.ispc:1]: fixed

File: tests/lit-tests/1238.ispc (hunk 2 above)
Comment (nurmukhametov) [reply tests/lit-tests/1238.ispc:1]: I have reduced it.

**Review by aneshlya — APPROVED** [review]
Looks good!

**Pull request merged**